Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -543,6 +537,10 @@ - (void)sidebarContainerDidHideSidebar:(SPSidebarContainerViewController *)sideb
     self.searchBar.userInteractionEnabled = YES;
 
     self.navigationController.navigationBar.userInteractionEnabled = YES;
+
+    if (!self.isSearchActive) {
```

Comment:
Nipticky: Mind adding a comment on this one? (I had to look it up why we'd set the VC as the first responder).

Just in case, to prevent someone in the future from deleting this and breaking shortcuts! 😄

Reply:
Good point!